sound/soc/intel/boards/Kconfig

```diff
@@ -498,4 +498,22 @@ config SND_SOC_INTEL_TGL_RT1308_MACH
 
 endif ## SND_SOC_SOF_TIGERLAKE
 
+if SND_SOC_SOF_JASPERLAKE
+
+config SND_SOC_INTEL_JSL_DA7219_MAX98373_MACH
+	tristate "JSL with DA7219 and MAX98373 in I2S Mode"
+	depends on I2C && ACPI
+	depends on MFD_INTEL_LPSS || COMPILE_TEST
+	select SND_SOC_DA7219
+	select SND_SOC_MAX98373
+	select SND_SOC_DMIC
+	select SND_SOC_HDAC_HDMI if SND_SOC_SOF_HDA_LINK
```

Comment:
@yongzhi1 oops, line510 should be replaced with:
select SND_HDA_CODEC_HDMI if SND_SOC_SOF_HDA_CODEC

Reply:
Thanks for catching it, updated https://github.com/yongzhi1/linux-1/commit/fd2cfb741c28a129f583b0f9065e8d7e01739467